tests/functional/c/comparison_with_callable_typing_constants.py

```diff
@@ -0,0 +1,19 @@
+"""
+Typing constants are actually implemented as functions, but they
```

Comment:
```suggestion
"""Typing constants are actually implemented as functions, but they
```

Well documented tests, thanks!